nsis/SharedDefs.nsh

```diff
@@ -62,4 +64,58 @@ ${If} $UserPrivileges == "Admin"
     ${ElseIf} $UserPrivileges == "User"
     StrCpy $IsAdmin 0
 ${EndIf}
+FunctionEnd
+
+Function ShowHelpMessage
+  !define line1 "Command line options:$\r$\n$\r$\n"
+	!define line2 "/S - silent install (must be uppercase)$\r$\n"
```

Comment:
Use spaces for indentation as the rest of the file, formatting could be a bit better too.

Reply:
How does it look now?